src/js/chartAuthoring.js

```diff
@@ -123,6 +153,24 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
                                     "onPieChartReady.escalate": "{chartAuthoring}.events.onPieChartReady.fire"
                                 }
                             }
+                        },
+
+                        sonifier: {
+                            type: "floe.chartAuthoring.sonifier",
+                            createOnEvent: "{chartAuthoring}.events.onPieChartReady",
+                            options: {
+                                model: {
+                                    "dataSet": "{dataEntryPanel}.model.dataSet",
+                                    // We relay currentlyPlayingData and isPlaying to the overall component so that sonification play events can be used to change the rest of the interface
+                                    currentlyPlayingData: "{chartAuthoring}.model.currentlyPlayingData",
```

Comment:
Is the model relay for ```currentlyPlayingData``` and ```isPlaying``` necessary? ```isPlaying``` seems only used in ```sonifier``` component. ```currentlyPlayingData``` is used by model relays for ```pie``` and ```legend``` components that can directly refer to ```{sonifier}.model. currentlyPlayingData```.